CHANGELOG.md

```diff
@@ -3,6 +3,120 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## [4.0.0-alpha.29](https://github.com/mui-org/material-ui-x/compare/v4.0.0-alpha.28...v4.0.0-alpha.29)
+
+_May 18, 2021_
+
+Big thanks to the 11 contributors who made this release possible. Here are some highlights ✨:
+
+- 🚀 Performance increased when filtering, sorting, and rendering (#1513) @dtassone
+- 💅 Add `columnHeader`, `row` and `cell` to the `classes` prop (#1660) @DanailH
+- ✅ Add `isRowSelectable` prop (#1659) @m4theushw
+- ⚡️ Add new icon slot to be displayed when the column is unsorted (#1415) @m4theushw
```

Comment:
Here and the line above. If we have a demo in the documentation, it would be great to link it. It makes it easier for developers to see what's new and either it could solve a pain for them or not.

Reply:
I linked to the demo for `isRowSelectable`. For the new icon slot we don't any demo. A gif would not communicate clearly because the current icon is subtle.